packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md

```diff
@@ -280,14 +280,15 @@ The OpenID Connect specification requires a `./well-known/openid-configuration`
 https://<id.domain.name>/oauth2/default/.well-known/openid-configuration
 ```
 
-You might notice that it has your Okta `dev-*` domain, rather than your custom domain name.
+You might notice that the URL has your Okta `dev-*` domain, rather than your custom domain name.
 
-You need to update your authorization server to use your custom domain to fix this:
+To fix this, update your authorization server to use your custom domain:
 
-1. Sign in to your Okta account and go to **API** > **Authorization Servers**.
-2. Select the  **default** custom authorization server, and then click **Edit**.
-3. Change the **Issuer** to use **Custom URL**.
-4. Try `./well-known/openid-configuration` again. It should now display your custom domain.
+1. Sign in to your Okta account and go to **Security** > **API** > **Authorization Servers**.
+1. Locate the  **default** custom authorization server, and then click **Edit**.
+1. In the **Settings** section, click **Edit**.
+1. Change the **Issuer** to **Custom URL** or **Dynamic (based on request domain)** (if enabled).
```

Comment:
Can we just have the recommendation be Dynamic, rather than mentioning the Custom URL option? In general Dynamic should be the guidance we give moving forward.

Reply:
Also was the numbering change intentional?

Reply:
Sure, I can make that change no problem.

The numbering change is intentional. VuePress renders the `1s` as an ordered list. It helps avoid mis-numbering that results from manual errors.